test/options-util.test.js

```diff
@@ -605,7 +605,7 @@ function createOptionsUpdatedTestData1() {
             }
         ],
         profileCurrent: 0,
-        version: 32,
+        version: 33,
```

Comment:
Those arent really needed anymore since we have tests for the cumulative result. It just shows that the replacing is correct between a specific region on specific versions. I wouldn't block a pr for it not including that.